js/pages/questionnaire.js

```diff
@@ -359,17 +359,17 @@ function externalListeners(){
 
 //BUILDING SOCCER
 function buildHTML(soccerResults, question) {
-    
-    let responseElement = question.querySelector("div[class='response']");
+    let fieldset = question.querySelector('fieldset');
+    let responseElement = fieldset.querySelector("div[class='response']");
     
     if (responseElement) {
       let tmp = responseElement.cloneNode(false);
-      question.replaceChild(tmp, responseElement);
+      fieldset.replaceChild(tmp, responseElement);
```

Comment:
I'm thinking line 367 may not be needed, since `responseElement` only need to point to the newly created node (line 368) and everything else stays the same. I may be wrong.

Reply:
Good idea, thanks. This chunk of code can be more direct. Updated.